model-optimizer/extensions/front/kaldi/replace_lstm_nonlinearity.py

```diff
@@ -40,11 +40,9 @@ def run_before(self):
 
     def replace_op(self, graph: Graph, node: Node):
         # split input to (i_part, f_part, c_part, o_part, ct_1)
-        split_node_axis = Const(graph, {'value': np.int64(1)}).create_node()
-        split_node = Split(graph, {'name': 'Split_lstm_input_',
-                                   'num_splits': 5}).create_node()
+        split_node = create_op_with_const_inputs(graph, Split, {1: np.int64(1)}, {'name': 'Split_lstm_input_',
```

Comment:
Change node name to `node.soft_get('name', node.id) + '/split_lstm_input'`

Reply:
Done.